lib/virt_autotest/utils.pm

```diff
@@ -192,10 +192,11 @@ sub create_guest {
         $virtinstall .= " --events on_reboot=$on_reboot" unless ($on_reboot eq '');
         $virtinstall .= " --extra-args '$extra_args'" unless ($extra_args eq '');
         record_info("$name", "Creating $name guests:\n$virtinstall");
-        script_run "$virtinstall >> ~/virt-install_$name.txt 2>&1 & true";    # true required because & terminator is not allowed
-
-        # wait for initrd to ensure the installation is starting
-        script_retry("grep -B99 -A99 'initrd' ~/virt-install_$name.txt", delay => 15, retry => 12, die => 0);
+        # HOTFIX: Run installation sequentially and retry it due to poo#101256
+        script_retry("$virtinstall >> ~/virt-install_$name.txt 2>&1", retry => 3, timeout => 2700);
```

Comment:
Will the `script_retry` work? You can't have two VMs with the same name / MAC address.

Reply:
I think a failed `virt-install` will delete the old virtual machine, so there is no remnant.
My doubts are actually that `virt-install` will never fail, so the script_retry will never retry.

Don't know, I'd like to try it out this way and re-evaluate after a couple of months.

Reply:
I think that:
a) The `virt-install` can never fail altho the installation stucked
b) If the `virt-install` fails it will not remove the virtual machine

Reply:
But please don't count this comment as a blocker.

Reply:
Let's turn then the approach around. I replace it with `assert_script_run` and if we see that it fails, then we can replace it with `script_retry`.